modules/admin-bar/assets/js/frontend/module.js

```diff
@@ -48,9 +50,13 @@ class AdminBar extends elementorModules.ViewModule {
 	 * @param adminBarConfig
 	 */
 	createMenu( adminBarConfig ) {
-		this.elements.$adminBar.append(
-			this.createMenuItems( Object.values( adminBarConfig ) )
-		);
+		const $items = this.createMenuItems( Object.values( adminBarConfig ) );
+
+		if ( this.elements.$editPageMenuItem.length ) {
+			this.elements.$editPageMenuItem.after( $items );
+		} else {
+			this.elements.$adminBar.append( $items );
```

Comment:
In this case it needs to be after `wp-admin-bar-new-content` + Explain it (e.g. archive pages. )